File: Linked-List/2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        long long positive_count = 0, negative_count = 0;
        for (int i = 0; i < size(nums); ++i) {
            if (nums[i] > 0) {
                ++positive_count;
            } else if (nums[i] < 0) {
                ++negative_count;
            }
        }
        return max(positive_count, negative_count);
    }
};

File: Linked-List/2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int pos = (int) Arrays.stream(nums).filter(x -> x > 0).count();
        int neg = (int) Arrays.stream(nums).filter(x -> x < 0).count();

        return Math.max(pos, neg);
    }
}
